### PR TITLE
sysutils/bind_exporter: New port: Prometheus exporter for BIND statistics

### PR DESCRIPTION
**Pull request by yzgyyang**
PR:		255449
Approved by:	lwhsu (mentor)

### CODE REVIEW

**Comment by lwhsu** [comment]
Most *_exporter ports are in net-mgmt category, I suggest either putting there or just putting to dns.

File: sysutils/bind_exporter/files/bind_exporter.in [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# bind_exporter_listen_address (string):Set ip:port that bind_exporter will listen on
+#               Default is ":9119".
+# bind_exporter_stats_url (string):     Set URL where BIND is serving statistics
+#               Default is "http://localhost:8053/".
+# bind_exporter_named_pidfile (path):   Set path to named's pidfile
+#               Default is "/var/run/named/pid".
+
+. /etc/rc.subr
+
+name=bind_exporter
```
Comment (lwhsu) [inline sysutils/bind_exporter/files/bind_exporter.in:27]: It's better to have a `desc` for `service bind_exporter describe`

**Comment by yzgyyang** [comment]
> Most *_exporter ports are in net-mgmt category, I suggest either putting there or just putting to dns.

@lwhsu Ah! I just checked node_exporter (in sysutils) since IMO it is the dominant one. No objection to move to net-mgmt though, will update the PR.

**Review by lwhsu — APPROVED** [review]
Others look good to me.

**Comment by yzgyyang** [comment]
Superseded by #69 with the approval from here.

**Pull request closed without merging**